lib/services/dnsManagement2/README.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Azure SDK for Node.js - DNS Management    
+    
+This project provides a Node.js package that makes it easy to manage Microsoft Azure DNS.     
+    
+## How to Install    
+    
+```bash    
+npm install azure-arm-dns    
+```  
```

Comment:
Please provide samples on CRUD. Please take a look at [README of storagemanagement](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/storageManagement2/README.md#how-to-use).